idd/Energy+.idd.in

```diff
@@ -83375,10 +83410,12 @@ Curve:CubicLinear,
   A2,  \field Input Unit Type for X
        \type choice
        \key Dimensionless
+       \key Temperature
```

Comment:
There must have been something that prompted you to add this, but I'm removing at for now since I cannot find a test file that uses this option.  In theory, all of the curve types should be OK with any kind of units, but this particular curve is a special type that's typically only used in ice storage.  But it is part of the BiVariateCurves group, so someone *could* apply it differently.  Since these are optional fields, will just leave as-is (just Dimensionless).  Rambling.